src/components/timeline/TrackMemoryGraph.js

```diff
@@ -220,7 +228,7 @@ class TrackMemoryGraphImpl extends React.PureComponent<Props, State> {
     const { width, rangeStart, rangeEnd, counter, interval } = this.props;
     const rangeLength = rangeEnd - rangeStart;
     const timeAtMouse = rangeStart + ((mouseX - left) / width) * rangeLength;
-    const { samples } = counter.sampleGroups;
+    const { samples } = counter.sampleGroups[0];
```

Comment:
Like above I think we should nicely throw if sampleGroups has no item. (we'll throw anyway, but better have a nice error than a dirty one)